frontend/src/Editor/AppVersionsManager.jsx

```diff
@@ -304,6 +304,11 @@ const CreateVersionModal = function CreateVersionModal({
             options={options}
             defaultValue={options[options.length - 1]}
             onChange={(version) => {
+              // If the changed value is a simple string, find the version
```

Comment:
Hey @dlebech, we can fix this by changing the value of the dropdown selection to be the version object rather than the version name value which is causing this bug.

https://github.com/ToolJet/ToolJet/blob/develop/frontend/src/Editor/AppVersionsManager.jsx#L246

In that `return` we can change it as 
``` return { ...version, label: version.name, value: version };``` 
Could you please make that change?

Reply:
Thanks @akshaysasidrn , beautiful solution. I've updated the PR.

Reply:
I have rebased the PR as well.